chapters/ch07.asciidoc

```diff
@@ -1096,7 +1091,7 @@ Object.keys(`🐎👱❤`)
 // <- [`0`, `1`, `2`, `3`, `4`]
 ----
 
-If we now consider a +for+ loop, we can observe more clearly how this is a problem. In the following example, we wanted to exfill each individual emoji from the +text+ string, but we get each code point instead.
+If we now consider a +for+ loop, we can observe more clearly how this is a problem. In the following example, we wanted to exfill each individual emoji from the +text+ string, but we got each code unit instead of the code points they form.
```

Comment:
Yeah it was a fancy way of saying "exfiltrate", but just noted the correct spelling is `exfil` plus many won't really understand the sentence 😅 extract it is